### ISSUE TITLE
Workrave doesn't detect idle time when gamepad is connected to USB port

### ISSUE THREAD

**Issue by MichaelOmegatron** [rcaelers/workrave#245]
**Describe the bug**
Workrave fails to detect idle time when my Playstation DualShock4 controller is plugged into a usb slot. I don't know if this is a DS4 specific problem, or if it's with any gamepad. This is not a problem with my keyboard or mouse. I submitted a previous bug report which might be related to this issue: https://github.com/rcaelers/workrave/issues/229. 

**To Reproduce**
1. Start Workrave with DS4 controller plugged in (or maybe any other gamepad too). Can also plug in after starting workrave.
2. Don't input any commands on any peripherals.
3. Workrave will not detect idle time. Active timer will continue to count down. Break timer will not count down.

**Expected behavior**
Workrave to detect idle time if no inputs are being presented regardless of what peripherals are plugged in.

** Windows (please complete the following information in case you encountered the bug on Windows):
 - Windows Version: [Windows 10 Pro OS build: 19041.572]
 - Workrave Version [e.g. 1.10.44]

**Additional context**

**Comment by rcaelers** [comment]
Indeed. I can reproduce this on my system. It seems many people complain about their computer not going to sleep when a game controller is connected. So it not just Workrave that suffers from this.

In the preferences, in the timers section, monitoring tab. Could you try to enable the 'Use alternate monitor' checkbox and check if that solves the issue? (You need to restart Workrave after changing this setting)

**Comment by MichaelOmegatron** [comment]
Just tried alternate monitoring, and it seemed to do the trick. I tried it before but didn't know I had to restart workrave. So I guess that fixes the problem. Thanks!
